TicTacToe.js

```diff
@@ -0,0 +1,320 @@
+//Readline bloiler plate code
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    readlineInterface.question(questionText, resolve);
+  });
+}
+
+let playerStatus = 1;
+rowOne = [' 1','|','2','|','3'];
+rowDashes = ['---','---','---'];
+rowTwo = [' 4','|','5','|','6'];
```

Comment:
Here you are storing the dividers (`|`) and indexes (`5`) inside the board arrays, which makes your `printBoard` function very simple, since all it has to do is `join` the rows. But it makes later code more complicated, or at least harder to understand -- `rowOne[2]` is the second cell, but `rowOne[4]` is actually the third cell, etc..  

But since the dividers (and spaces and indexes) are only used inside of `printBoard`, you could simplify your data structure and the rest of the program to only store the values of what is currently *inside* each cell, and put some more logic *hidden inside printBoard* for printing dividers etc.

What would happen if instead of `rowOne`, `rowTwo`, and `rowThree`, you just had a single array, `let board = []`, and set X or O inside that array when a move is made?